### DISCUSSION TITLE
Cannot select a specific version of GHES to deploy.

### DISCUSSION THREAD

**Discussion by zhouwenyang** [community/community#38451] [Enterprise]
help, I meet some problem during setup github replica. I cannot select a specific version of GHES to deploy.
I want to deploy GHES 3.6.2, but it's always 3.6.3, anywhere I can change the version?

<img width="1239" alt="image" src="https://user-images.githubusercontent.com/6780666/199386249-8c25a372-f74e-4ce0-abf4-6dacf8b212a7.png">

**Comment by djdefi** [comment]
The source of the Azure template for GitHub Enterprise is located in this Azure maintained repository: https://github.com/Azure/azure-quickstart-templates/tree/master/application-workloads/github-enterprise/github-enterprise 

You could edit this to use a different release or instance configuration. The deployment template link from the releases page in your screenshot will install the latest release as we don't currently maintain multiple versions of the template. 

Alternatively, to install an older release another option is using the `az` CLI option described in this guide: https://docs.github.com/enterprise-server/admin/installation/setting-up-a-github-enterprise-server-instance/installing-github-enterprise-server-on-azure#creating-the-github-enterprise-server-virtual-machine
